RapidRiverSoftware/rapidriversoftware.github.io
# Page | Case studies

**This PR includes:**
- a new Case Studies page design
- case study content and illustrations
- burger menu on the left
- a downgrade of the Jekyll image version
- updated hire us CTA design at bottom of case studies and services page
- a fix for horizontal scroll bug on hire us page
- a bunch of small UI fixes

**To test:**
- In your browser, resize width (or use Responsive Design Mode)
- make sure nothing is broken on small, medium, large screens
